src/source.rs

```diff
@@ -103,65 +115,45 @@ impl FileSource {
         )
     }
 
-    pub fn fetch_file_sync(&self, langid: &LanguageIdentifier, path: &str) -> ResourceOption {
+    pub fn fetch_file_sync(&self, langid: &LanguageIdentifier, path: &str) -> RcResourceOption {
+        use ResourceStatus::*;
+
         let full_path = self.get_path(langid, &path);
 
-        let mut cache = self.cache.try_borrow_mut().unwrap();
-        let res = cache.entry(full_path.clone()).or_insert_with(|| {
-            (self.fetch_sync)(&full_path)
-                .expect("I/O Error")
+        // println!("[l10nregistry] fetch_file_sync: {:?}", full_path);
```

Comment:
leftover? or do you want to leave it as `debug`?

Reply:
I've converted to `trace!` macros.